pkg/metric_storage/metric_storage.go

```diff
@@ -374,6 +375,7 @@ func (m *MetricStorage) ApplyOperation(op operation.MetricOperation, commonLabel
 		m.GaugeSet(op.Name, *op.Value, labels)
 		return
 	}
+	// nolint:staticcheck
```

Comment:
Why do we need to ignore the staticcheck here? Maybe we can add this case to the global exclude settings instead?

Reply:
Because Add and Set properties for MetricOperation were marked as Deprecated and staticcheck linter does not allow to use them anymore. Thats why old code marked as nolint. Don't think that its a good idea to make a global exclude for this case